spec/unit/messages/users_list_message_spec.rb

```diff
@@ -35,6 +35,21 @@ module VCAP::CloudController
         expect(message.requested?(:origins)).to be_truthy
       end
 
+      context 'partial_usernames is used in .from_params' do
```

Comment:
we can add the `partial_usernames` into the existing `params` at the root but the UsersListMessage constructed with all the parameters wont be valid as `usernames` and `partial_usernames` cannot coexist together. That's why we decided to move it into a different context. We can try to modify the test so that we add a before block were we modify `params`.

Reply:
Also, other contexts (i.e. `guids, usernames, origins are nil` and `guids, usernames, origins must be arrays`) are using a similar pattern as we are. Perhaps the context naming we selected is not the best.